### DISCUSSION TITLE
Read Ahead hard

### DISCUSSION THREAD

**Discussion by frogfrogfrogfrogfrog** [monkeytypegame/monkeytype#876] [Ideas] [answered]
it's self explanatory

instead of 2 words make it 3 

since read ahead has an easy mode why can't it have a hard mode?

**Comment by Miodec** [comment, marked as answer]
Alright

**Comment by frogfrogfrogfrogfrog** [comment]
It seems this feature will never be added after all...

**Reply by Miodec** [reply]
Im a forgetful guy. Ill add an issue so i dont forget anymore, sorry!

**Reply by frogfrogfrogfrogfrog** [reply]
woohoo
